addons/block_code/ui/blocks/utilities/parameter_input/parameter_input.gd

```diff
@@ -126,10 +131,16 @@ func get_string() -> String:
 			return "%s" % input
 
 
-func _on_line_edit_text_changed(new_text):
+func _on_line_edit_text_submitted(new_text):
```

Comment:
This function is also used to handle the Vector2 input signals. You'll need to split those to a separate function (arguably they should have been already).

Reply:
Oh I totally missed that, good catch! PR updated. It is a bit boilerplate now. An alternative would be to connect to the callbacks in code (not in the Signals dock in the editor) and bind the corresponding LineEdit as argument.